src/spicy/spicyz/glue-compiler.h

```diff
@@ -269,8 +282,7 @@ class GlueCompiler {
 
     std::map<hilti::ID, std::shared_ptr<glue::SpicyModule>> _spicy_modules;
 
-    std::vector<std::pair<hilti::ID, std::optional<hilti::ID>>>
-        _imports;                                            /**< imports from EVT files, with ID and optional scope */
+    std::vector<std::pair<hilti::ID, hilti::ID>> _imports;   /**< imports from EVT files, with ID and optional scope */
```

Comment:
The scope is not optional anymore so let's update the docstring, e.g.,

```suggestion
    std::vector<std::pair<hilti::ID, hilti::ID>> _imports;   /**< imports from EVT files, with ID and scope */
```

Reply:
Well, it is still optional but that would now be an empty ID. I'll clarify the comment.